### PR TITLE
[BEAM-2297] support UNION

### PR DESCRIPTION
**Pull request by xumingming**
support UNION

### CODE REVIEW

**Comment by xumingming** [comment]
R: @XuMingmin

File: dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/BeamPlannerAggregationSubmitTest.java [hunk 1]
```diff
@@ -41,10 +42,11 @@
   public static DateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
   public static BeamSqlRunner runner = new BeamSqlRunner();
 
-  @BeforeClass
```
Comment (XuMingmin) [inline dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/BeamPlannerAggregationSubmitTest.java]: Any thoughts to replace with `Before`? I suppose the `prepare()` only need to run once, which create the initial tables for all test cases.

File: dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/BeamPlannerSubmitTest.java [hunk 2]
```diff
@@ -19,13 +19,19 @@
...
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
 /**
  * Tests to execute a query.
  *
  */
 public class BeamPlannerSubmitTest extends BasePlanner {
+  @Before
```
Comment (XuMingmin) [inline dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/BeamPlannerSubmitTest.java]: change to `After` as a clean-up step?

**Comment by xumingming** [comment]
Closing this one, will be merged into PR: https://github.com/apache/beam/pull/3183 to enable more code reuse.

**Pull request closed without merging**